spec/requests/black_listed_resources_spec.rb

```diff
@@ -3,10 +3,8 @@
      partners-uc-landlords
      resources-for-professionals-working-with-young-people-and-parents).each do |category|
     context "request category #{category}" do
-      it 'returns a 501 response' do
-        get("/en/categories/#{category}")
-
-        expect(response.status).to eq(501)
+      it 'returns a 404 response' do
+        expect{ get "/en/categories/#{category}" }.to raise_error(ActionController::RoutingError)
```

Comment:
```
context "request category #{category}" do
      it 'returns a 404 response' do
        get "/en/categories/#{category}"
        expect{ get "/en/categories/#{category}" }.to raise_error(ActionController::RoutingError)
        expect(response.status).to be(404)
      end
    end
```

@tomas-stefano 
This blows up because ```get "/en/categories/#{category}"``` throws an exception.